scheduler/src/cook/kubernetes/controller.clj

```diff
@@ -152,6 +153,18 @@
                                      @(:pool-name->fenzo-state-atom compute-cluster)
                                      mesos-status))
 
+(defn write-status-to-cook-passport
```

Comment:
Maybe just `write-status-to-passport` (since Cook is implied)?

Reply:
Sounds good to me